### PR TITLE
add twitter info for Sens. Cassidy, Smith, and Jones

### PR DESCRIPTION
**Pull request by jeveleth**

### CODE REVIEW

File: legislators-social-media.yaml [hunk 1]
```diff
@@ -1879,6 +1879,11 @@
     youtube_id: UCqMici9I7nz_i-PN7eSYjwQ
     twitter_id: 24745957
     instagram: rephankjohnson
+- id:
+    bioguide: J000300
+    govtrack: 412741
+  social:
+    twitter: sendougjones
```
Comment (konklone) [inline legislators-social-media.yaml]: Doug Jones' official site isn't set up yet, but the Verified mark, and the bio/link on the Twitter profile that goes to the official site, makes me comfortable this isn't a campaign site. :+1:

File: legislators-social-media.yaml [hunk 2]
```diff
@@ -3995,6 +4000,7 @@
     thomas: '01925'
     govtrack: 412269
   social:
+    twitter: BillCassidy
```
Comment (konklone) [inline legislators-social-media.yaml]: This is a campaign account and links to billcassidy.com from their bio. I don't see an official account linked from https://www.cassidy.senate.gov. Please take this one out of the PR.

File: legislators-social-media.yaml [hunk 3]
```diff
@@ -4189,6 +4195,11 @@
     youtube: RepSanfordSC
     youtube_id: UCS33edNAxXe2pyFbnUUcZWA
     twitter_id: 1499993378
+- id:
+    bioguide: S001203
+    govtrack: 412742
+  social:
+    twitter: SenTinaSmith
```
Comment (konklone) [inline legislators-social-media.yaml]: Sen. Smith doesn't have an official site yet, but the Verified checkmark, the bio, and the newness of the account are enough for me to accept this as the Senator's official account. :+1:

File: legislators-social-media.yaml (hunk 2 above)
Comment (jeveleth) [reply legislators-social-media.yaml]: Sure thing. I've now removed it.

**Comment by konklone** [comment]
Thank you!

**Pull request merged**